CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## main / unreleased
 
+* [FEATURE] Extend distributor trace logger with optional features to include span attributes and filter by error status. [#1465](https://github.com/grafana/tempo/pull/1465) (@faustodavid)
```

Comment:
We should note the breaking change/deprecation here.

Reply:
done. 

Should I add a warning log if the deprecated flag is used?
https://github.com/grafana/tempo/blob/main/cmd/tempo/app/app.go#L146

Reply:
That'd be ideal yes. Thanks :)

Reply:
done :)